modules/logging/aws-for-fluent-bit/iam.tf

```diff
@@ -17,7 +17,7 @@ data "aws_iam_policy_document" "aws_for_fluent_bit_assume_role_policy" {
 }
 
 resource "aws_iam_role" "aws_for_fluent_bit_iam_role" {
-  name               = "${var.cluster_name}-aws-for-fluent-bit"
+  name               = "${var.cluster_name}-aws-for-fluent-bit-${var.environment}"
```

Comment:
cluster_name unique name, why you want set environment? I mean if you deploy cluster-dev and cluster-prod, first prefix enough

Reply:
Had an, as it appears, wrong idea that we combine env and cluster names into a unique identifier. Please decline that PR then